substrate/primitives/core/Cargo.toml

```diff
@@ -126,29 +123,22 @@ std = [
 
 # Serde support without relying on std features.
 serde = [
-	"array-bytes",
 	"blake2",
 	"bounded-collections/serde",
 	"bs58/alloc",
 	"dep:serde",
 	"impl-serde",
 	"primitive-types/serde_no_std",
 	"scale-info/serde",
-	"secrecy/alloc",
-	"sp-crypto-hashing",
 	"sp-storage/serde",
 ]
 
 # This feature enables all crypto primitives for `no_std` builds like microcontrollers
 # or Intel SGX.
 # For the regular wasm runtime builds this should not be used.
 full_crypto = [
-	"array-bytes",
 	"blake2",
-	"ed25519-zebra",
 	"libsecp256k1",
-	"secp256k1",
-	"sp-crypto-hashing",
 	"sp-runtime-interface/disable_target_static_assertions",
 ]
```

Comment:
This `full_crypto` feature at this point is quite pointless.

Almost any crypto we provide doesn't require on randomness except `sr25519`.

But nevermind, I can eventually go through it when moving this stuff to the crypto folder.
Each crypto primitive will be gated by its own feature so is also pointless removing `full_crypto` right now